README.md

```diff
@@ -303,32 +306,34 @@ Meters report consumption over the service.
 
 #### Service names
 
-Service name    | Units                                     | Description
-----------------|-------------------------------------------|------------
-`meter_elec`    | kWh, kVAh, W, pulse_c, V, A, power_factor | Electric meter
-`meter_gas`     | cub_m, cub_f, pulse_c                     | Gas meter
-`meter_water`   | cub_m, cub_f, gallon, pulse_c             | Water meter
-`meter_heating` | kWh                                       | Heating meter
-`meter_cooling` | kWh                                       | Cooling meter
+Service name    | Measurements                                                                     | Description
+----------------|----------------------------------------------------------------------------------|------------
+`meter_elec`    | energy, power, reactive power, frequency, voltage, current, power_factor, pulses | Electric meters
+`meter_gas`     | volume, energy, pulses, flow, pressure                                           | Gas meter
+`meter_water`   | volume, energy, pulses, flow, pressure                                           | Water meter
+`meter_heating` | energy consumption                                                               | Heating meter
+`meter_cooling` | energy consumption                                                               | Cooling meter
 
 #### Interfaces
 
-Type | Interface                     | Value type | Properties                         | Description
------|-------------------------------|------------|------------------------------------|-------------
-in   | cmd.meter.get_report          | string     | direction                          | Value - is a unit. May not be supported by all meters.
-in   | cmd.meter.reset               | null       |                                    | Resets all historical readings.
-out  | evt.meter.report              | float      | unit, prv_data, delta_t, direction |
-out  | evt.meter_ext.report          | float_map  |                                    | [Extended meter report](#extended-report-object) with up to 17 data points
-in   | cmd.meter_ext.get_report      | null       |                                    | Request extended report
+Dir  | `type`                        | `val_t`            | `prop`                             | Description
+-----|-------------------------------|--------------------|------------------------------------|-------------
+in   | cmd.meter.get_report          | string             | direction                          | Triggers evt.meter.report. An argument is "export" or "import".
```

Comment:
This is actually not true.
If by an "argument" You mean a "val", then an argument is a unit.